src/TheMovie/TheMovie/App.xaml.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Prism;
+using Prism.Autofac;
+using Prism.Ioc;
+using TheMovie.Services;
+using TheMovie.ViewModels;
+using TheMovie.Views;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+[assembly: XamlCompilation(XamlCompilationOptions.Compile)]
+namespace TheMovie
+{
+    public partial class App : PrismApplication
+    {
+        public App() : this(null) { }
+
+        public App(IPlatformInitializer initializer) : base(initializer) { }
+
+        protected override async void OnInitialized()
```

Comment:
What problems can you face with `async void` methods and how do you typically fix/avoid them?

Reply:
Any unhandled exception will finalize the process and I believe we can do not wait for the completion of the method. We can fix it by changing the void for Task.

Reply:
Yeah, right. We usually try-catch all the `async void` methods, or use Fody extension, or turn methods into `Task` signature and handle exceptions above.